### PR TITLE
Add describing of "pixel" style.

### PR DESCRIPTION
**Pull request by Aeliot-Tm**
Closes https://github.com/antonkomarev/github-profile-views-counter/issues/95

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,12 +87,13 @@ You can use any valid HEX color or pick from a predefined set of named colors (`
...
-| `flat-square` | ![](https://img.shields.io/static/v1?label=Profile+views&message=1234567890&color=007ec6&style=flat-square) |
-| `plastic` | ![](https://img.shields.io/static/v1?label=Profile+views&message=1234567890&color=007ec6&style=plastic) |
+| style           | demo                                                                                                          |
+|-----------------|---------------------------------------------------------------------------------------------------------------|
+| `flat`          | ![](https://img.shields.io/static/v1?label=Profile+views&message=1234567890&color=007ec6&style=flat)          |
+| `flat-square`   | ![](https://img.shields.io/static/v1?label=Profile+views&message=1234567890&color=007ec6&style=flat-square)   |
+| `plastic`       | ![](https://img.shields.io/static/v1?label=Profile+views&message=1234567890&color=007ec6&style=plastic)       |
 | `for-the-badge` | ![](https://img.shields.io/static/v1?label=Profile+views&message=1234567890&color=007ec6&style=for-the-badge) |
+| `pixel`         | this is "invisible" mode for counter. Use it when you want to have counter without of displaying of it        |
```
Comment (antonkomarev) [inline README.md:96]: I'd prefer to not add spaces between columns. Otherwise it adds many changes to other rows.

**Review by antonkomarev — CHANGES_REQUESTED** [review]

**Comment by antonkomarev** [comment]
Thanks for the contribution! @Aeliot-Tm

File: README.md (hunk 1 above)
Comment (Aeliot-Tm) [reply README.md:96]: Ok. I'll revert it today later

Comment (Aeliot-Tm) [reply README.md]: Hi @antonkomarev,

Fixed. Could you have a look?

**Review by antonkomarev — APPROVED** [review]

**Pull request merged**